.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest a new feature or changes to existing features 
+
+---
+
+## Feature Request Type
+
+- [ ] Core Functionality
+- [ ] Add-on hardware support (eg audio, rgb, oled screen, etc)
+- [ ] Alteration (enhancement/optimization) of existing Feature(s)
+- [ ] New behavior
+
+## Description 
+A few sentances describing what it is that you'd like to see. Additional information (such as links to spec sheets, licensing info, other related issues or PR's, etc) would be helpful. 
```

Comment:
While the template is nice for us, I worry it may intimidate people who may have never filed a bug or wandered into Github before. Perhaps we should put in some verbage saying that its optional?

Reply:
Absolutely. Maybe we should add a blank issue, as well?